models/engine/file_storage.py

```diff
@@ -68,3 +68,12 @@ def delete(self, obj=None):
     def close(self):
         """call reload() method for deserializing the JSON file to objects"""
         self.reload()
+
+    def get(self, cls, id):
+        """get the obejects by id, using dict's get()"""
+        if cls in classes and id and type(id) == str:
+            return self.__objects.get(cls + '.' + id, None)
```

Comment:
Similarly, if we are matching the test program and being fed a `cls` that is a `State` object and not the string `'State'`, then we need to change this to `cls.__name__ + '.' + id`.

Also, I like the readability of `+` concatenation, but have been converted to the gospel of `join()`. :)